packages/hash/playwright/tests/utils/getDerivedPayloadFromMostRecentEmail.ts

```diff
@@ -31,9 +31,11 @@ const waitForRecentFileChange = async (
     await sleep(checkIntervalInMs);
     remainingWaitTime -= checkIntervalInMs;
   } while (remainingWaitTime > 0);
-
+  const timeLastEdited = (await fs.stat(filePath)).mtimeMs;
```

Comment:
This statement may throw if `filePath` does not exist. I guess we can revert this file as it looks unrelated.